src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link, useParams } from 'react-router-dom';
+import { DateTime } from 'luxon';
+
+import BreadcrumbNav from 'components/BreadcrumbNav';
+import Footer from 'components/Footer';
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+import PageWrapper from 'components/PageWrapper';
+import cmsDivisionsAndOffices from 'constants/enums/cmsDivisionsAndOffices';
+import { AppState } from 'reducers/rootReducer';
+import { fetchSystemIntake } from 'types/routines';
+
+import './index.scss';
+
+const GovernanceReviewTeam = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const { systemId } = useParams();
+
+  useEffect(() => {
+    dispatch(fetchSystemIntake(systemId));
+  }, [dispatch, systemId]);
+
+  const systemIntake = useSelector(
+    (state: AppState) => state.systemIntake.systemIntake
+  );
+
+  const component = cmsDivisionsAndOffices.find(
+    c => c.name === systemIntake.requester.component
+  );
+
+  const requesterNameAndComponent = component
+    ? `${systemIntake.requester.name}, ${component.acronym}`
+    : systemIntake.requester.name;
+
+  return (
+    <PageWrapper className="easi-grt">
+      <Header />
+      <MainContent>
+        <section className="easi-grt__request-summary">
+          <div className="grid-container padding-y-2">
+            <BreadcrumbNav>
```

Comment:
In the wireframe, it looks like the breadcrumb is supposed to be white. In my testing, it's showing up as purple (default browser visited link behavior). Should we apply a different style to the breadcrumb for this page? Make a new component. Not sure the best way to solve it.

Reply:
Yeah I"m not sure either, the purple color is default behavior for visited links. I'd want to stay on the cautious side and not override the color of visited links.